### ISSUE TITLE
ColumnExists query breaks when there are multiple river queues across different schemas

### ISSUE THREAD

**Issue by themaxgoldman** [riverqueue/river#503]
I am using a River setup where we have multiple river queues with the db for each located in a different schema. This was based on a discussion with one of my coworkers [here](https://github.com/riverqueue/river/discussions/343). When trying to run the most recent migrations the migration succeeded for the first schema/queue but then failed for the second and third schema. The error message was 
```
❯ river migrate-up --database-url "postgres://user:pass@localhost:1234/chariot?search_path=SCHEMA_NAME"
failed: error getting existing migrations for line "main": ERROR: column "line" does not exist (SQLSTATE 42703)
```
After digging into the code I found that the River CLI checks whether the "line" column exists is through [the ColumnExists query](https://github.com/riverqueue/river/blob/08ef6349316815d667e672d19b4fb389af464249/riverdriver/riverpgxv5/internal/dbsqlc/river_migration.sql#L71-L76).

The issue with this is it doesn't take into account the schema on the search path, so after one migration has run, the query will always return true. This caused [this code](https://github.com/riverqueue/river/blob/08ef6349316815d667e672d19b4fb389af464249/rivermigrate/river_migrate.go#L540-L551) to be skipped, which I assume was meant to be run when the line was missing and then we see the error above. 

I'm not sure why the query doesn't respect the search path we provide in the database url. I'm not sure if there's a workaround here - I'd appreciate any help!

**Comment by brandur** [comment]
Thanks for the report Max. Opened a fix at #505.

**Comment by themaxgoldman** [comment]
Great, thanks for fixing this so quickly!. We love River!

**Comment by brandur** [comment]
Excellent, thanks :)

Fix released in v0.11.1. Remember to `go install` the CLI `@latest`.
